### DISCUSSION TITLE
(QoL) Make follow points and hitcircle fadeout toggleable from the options menu.

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#15137] [Ideas] [answered]
Seeing as follow points and hitcircle fadeout are *generally* things people like to have on or off for all skins the option to have a global toggle like hit lighitng does would be much appreciated.

**Comment by integerrr** [comment]
hitcircle fadeout #15123 
hit lighting is not skinnable afaik that's why it makes sense to go in options, for skinnable stuff like follow points i'd much rather see a toggle (or achieved in other manners) IN skinning editor than in options

**Comment by peppy** [comment, marked as answer]
This will be implemented as part of skinning customisations, when we get around to adding those systems in.
